Description: Do not build static files in the Makefile, they are not used in Debian.
Author: Javier Fernandez-Sanguino <jfs@debian.org> for Debian
Forwarded: not-needed, this patch is Debian-specific

--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 LIBS=$(shell libgcrypt-config --libs)
 
 
-all: chntpw chntpw.static cpnt reged reged.static samusrgrp samusrgrp.static sampasswd sampasswd.static
+all: chntpw cpnt reged samusrgrp sampasswd
 
 chntpw: chntpw.o ntreg.o edlib.o libsam.o
 	$(CC) $(CFLAGS) -o chntpw chntpw.o ntreg.o edlib.o libsam.o $(LIBS)
@@ -52,5 +52,5 @@
 	$(CC) -c $(CFLAGS) $<
 
 clean:
-	rm -f *.o chntpw chntpw.static cpnt reged reged.static samusrgrp samusrgrp.static sampasswd sampasswd.static *~
+	rm -f *.o chntpw cpnt reged samusrgrp sampasswd *~
 
